File: Treinamento/C/Estrutura_de_decisão/Questão05.c
#include <stdio.h>

int main() {
    int n1, n2, n3;

    printf("Digite o primeiro número: ");
    scanf("%i", &n1);

    printf("Digite o segundo número: ");
    scanf("%i", &n2);

    printf("Digite o terceiro número: ");
    scanf("%i", &n3);

    if (n1 > n2 && n1 > n3) {
        printf("Maior número: %i\n", n1);
    } else if (n2 > n1 && n2 > n3) {
        printf("Maior número: %i\n", n2);
    } else if (n3 > n1 && n3 > n2) {
        printf("Maior número: %i\n", n3);
    }
}

File: Treinamento/Java/Estrutura_de_decisão/Questão05.java
import java.util.Scanner;
public class Questão05 {
    public static void main(String[] args) {
        Scanner s;
        s = new Scanner(System.in);

        System.out.print("Digite o primeiro número: ");
        int n1 = s.nextInt();

        System.out.print("Digite o segundo número: ");
        int n2 = s.nextInt();
        
        System.out.print("Digite o terceiro número: ");
        int n3 = s.nextInt();

        if (n1 > n2 && n1 > n3) {
            System.out.println("Maior número: " + n1);
        } else if (n2 > n1 && n2 > n3) {
            System.out.println("Maior número: " + n2);
        } else {
            System.out.println("Maior número: " + n3);
        }
    }
}
            
